sources/FILEIO

```diff
@@ -668,22 +666,29 @@ Copyright (c) 1981-1993, 1999, 2020-2022 by Venue & Xerox Corporation.
     T])
 
 (\STREAM.DEFPRINT
-  [LAMBDA (STRM OUTSTREAM)                               (* ; "Edited 19-Aug-88 14:01 by bvm")
+  [LAMBDA (STRM OUTSTREAM)                                   (* ; "Edited  9-Oct-2022 08:58 by lmm")
+                                                             (* ; "Edited 19-Aug-88 14:01 by bvm")
     (LET ((TYPE (SELECTC (fetch ACCESSBITS of STRM)
                     (ReadBit "Input")
                     (OutputBits "Output")
                     (BothBits "IO")
                     (AppendBit "Append")
                     "Closed")))
-         (\DEFPRINT.BY.NAME STRM OUTSTREAM NIL
-                (COND
-                   ((fetch (STREAM NAMEDP) of STRM)  (* ; "Use file name")
-                    (CONCAT TYPE " Stream on " (fetch (STREAM FULLFILENAME) of STRM)))
-                   (T                                        (* ; "Name the device")
-                      (CONCAT TYPE " " [CL:STRING-CAPITALIZE (STRING (fetch (FDEV DEVICENAME)
-                                                                        of (fetch DEVICE
-                                                                                  of STRM]
-                             " Stream"])
+         (\DEFPRINT.BY.NAME STRM OUTSTREAM NIL (COND
+                                                  ((fetch (STREAM NAMEDP) of STRM)
+                                                             (* ; "Use file name")
+                                                   (CONCAT TYPE " Stream on " (fetch (STREAM 
+                                                                                         FULLFILENAME
+                                                                                            )
+                                                                                 of STRM)))
+                                                  ((TYPE? FDEV (FETCH DEVICE OF STRM))
+                                                             (* ; "Name the device")
+                                                   (CONCAT TYPE " "
+                                                          [CL:STRING-CAPITALIZE
+                                                           (STRING (fetch (FDEV DEVICENAME)
+                                                                      of (fetch DEVICE of STRM]
+                                                          " Stream"))
+                                                  (T (CONCAT TYPE " unknown stream"])
```

Comment:
perhaps " unknown Stream" (capital S).  But is "unknown" the right word, or should it just be " Stream"?

Reply:
just Stream